sdk/hdinsight/Microsoft.Azure.HDInsight.Job/src/Customizations/HDInsightJobManagementClient.Customization.cs

```diff
@@ -39,9 +39,6 @@ public static RetryPolicy HDInsightRetryPolicy
         /// Required. The cluster dns name against which the job management is
         /// to be performed.
         /// </param>
-        /// <param name='userName'>
```

Comment:
Why was this removed?

Reply:
I assume some validation failed because this parameter does not exist?

Reply:
That was raising a warning. But I guess that wasn't an actual issue. I added it back.

Reply:
warnings will turn into errors in our CI builds so you should still clean this up. I just wanted to ensure I understood why you removed it.